upgrading/topics/migrate_db.adoc

```diff
@@ -32,6 +32,28 @@ Or run this CLI command:
 When you start the server with this setting your database is automatically migrated if the database schema has changed
 in the new version.
 
+To change the threshold (the number of records) for automated index creation, set the indexCreationThreshold property
```

Comment:
Added a description.

Reply:
Thank you, maybe we should add information that there will be warn message in logs with SQL command which can be then applied later manually in case the number of records is higher than the `indexCreationThreshold` property, wdyt?

Reply:
I see. Added the case.